lib/Common/CommonDefines.h

```diff
@@ -538,3 +538,32 @@
 #ifndef PROFILE_DICTIONARY
 #define PROFILE_DICTIONARY 0
 #endif
+
+// SIMD_JS
+#define SIMD_EXPAND_W_TAG(MACRO) \
```

Comment:
This might be a good place to add my if(func->HasSIMD()).  Just add func as an argument to the macro.